components/dependency_injection/compilation.rst

```diff
@@ -493,14 +493,14 @@ for these resources and use them as metadata for the cache::
     $containerConfigCache = new ConfigCache($file, $isDebug);
 
     if (!$containerConfigCache->isFresh()) {
-        $containerBuilder = new ContainerBuilder();
+        $container = new ContainerBuilder();
```

Comment:
Why that change?

Reply:
Becuase almost whole docs use `$container` and even https://github.com/symfony/dependency-injection/blob/master/Extension/Extension.php#L80 it is named as `$container`, no need to mix it.

Reply:
I disagree here. This is one of those cases where variable naming is extremely important. A container builder is not a container, so the variable name is wrong.

Reply:
So then `$container` in Extension is wrong also?

Reply:
Yep, it should be `$containerBuilder` as well